Makefile.lb

```diff
@@ -26,6 +26,10 @@ install: |checkout_deps
 	$(Q) cp -a config* $(SPDK_INSTALL_DIR)
 	$(Q) cp -a include $(SPDK_INSTALL_DIR)
 
+config_overwrite:
```

Comment:
@gillev-lb same question as in previous PR why it's not installed pre checkin to osmosis ?

Reply:
It is, but this is a full path such as :
/home/gillev/workspace1/build/dev/spdk-sdk/170a6ec76a6f
returned by the componenet-tool
So when I fetch from osmosis, it will be valid only if I'm the one to check-in.
If you fetch, you must change it.
I guess there might be better solution - like changing the componenet tool to return  a path such as
${SOME_BASIDE_DIR_EVERYONE_HAVE_IN_ENV}/dev/spdk-sdk/170a6ec76a6f 
and not an explicit one. 
Another option is to manipulate the string returned from componentr-tool and create that string.
My solution was faster.